src/App.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchCryptoData } from './redux/cryptoSlice';
+import CryptoList from './components/CryptoList';
+import CryptoDetails from './components/CryptoDetails';
+import Navbar from './components/Navbar';
+
+function App() {
+  const dispatch = useDispatch();
+
```

Comment:
- [Optional] great job with the design. Please as we know this was a mobile app, kindly ensure the  design stays right down to the mobile screen size of 480px. 
![Screenshot_30](https://github.com/CVILLA09/my-crypto-metrics/assets/65298048/af37aa60-45ee-4751-9542-34d240629b56)

Reply:
@tufoinnkuo10 

Thank you for the approval and constructive feedback. 
I'll work on improving audio and video quality in future presentations.

For your optional points:

- I'll add arrows to the cards.
- I'll ensure responsiveness down to 480px.

Your guidance is invaluable. Cheers! 🚀💻